pkg/cmd/server/kubernetes/node.go

```diff
@@ -232,7 +232,7 @@ func (c *NodeConfig) EnsureLocalQuota(nodeConfig configapi.NodeConfig) {
 		// Can't really do type checking or use a constant here as they are not exported:
 		if plugin.CanSupport(emptyDirSpec) {
 			wrapper := emptydir.EmptyDirQuotaPlugin{
-				Wrapped:         plugin,
+				VolumePlugin:    plugin,
```

Comment:
@dgoodwin if you can review these changes based on what has happened upstream to EmptyDir (and @pmorie too)